requirements.txt

```diff
@@ -1,73 +1,13 @@
-alembic==1.4.2
-aniso8601==3.0.0
-appdirs==1.4.3
-APScheduler==3.5.1
-asn1crypto==0.24.0
-attrs==19.3.0
-awsebcli==3.14.1
-bcrypt==3.1.7
-black==19.3b0
-blessed==1.15.0
+APScheduler==3.6.3
 blinker==1.4
```

Comment:
@SanketDG is this blinker useful??

Reply:
I don't found anywhere we are using it and maybe I forgot to remove it. Therefore, removing it now.